app/services/VatPartialBuilder.scala

```diff
@@ -124,7 +124,7 @@ class VatPartialBuilderImpl @Inject()(val enrolmentsStore: EnrolmentsStoreServic
       case _: VatNoEnrolment =>
         Future.successful(buildVatVarEnrolmentPrompt(forCard))
       case VatVarEnrolment(_, false) =>
-        enrolmentsStore.showNewPinLink(request.vatVarEnrolment, DateTime.now, request.credId).map(
+        enrolmentsStore.showNewPinLink(request.vatVarEnrolment, LocalDateTime.now, request.credId).map(
```

Comment:
Following on from the enrolment service this should be OffsetDateTime.now() to handle GMT/BST.